src/components/Explore/Explore.style.ts

```diff
@@ -14,6 +14,12 @@ function palette(props: any) {
   return props.theme.palette.chart;
 }
 
+/** By default make line charts solid,
+ * If a value is provided, use that instead.*/
+function getStrokeDashType(props: any) {
```

Comment:
I  tried to make this inline (without declaring a named function) but I got a type error b/c `props.strokeDashArray` had a type of `any`. Not sure if this is significant but I just wanted to note it